jekyll/_cci2/api-intro.md

```diff
@@ -0,0 +1,122 @@
+
+**IMPORTANT** Pipeline parameters are **not** treated as sensitive data and **must not** be used for sensitive values (secrets). You can find this sensitive information in the [Project Settings] page(https://circleci.com/docs/2.0/settings/) and [Contexts](https://circleci.com/docs/2.0/glossary/#context)
+
+## Changes In Endpoints
+
+The CircleCI API v2 release includes several new endpoints, as well as some other endpoints that have been deprecated. The sections below list the endpoints added for this release, in addition to the endpoints that have been removed.
+
+For a complete list of all API v2 enpoints, please refer to the [API v2 Reference Guide](https://circleci.com/docs/api/v2/#circleci-api), which contains a detailed description of each individual endpoint, as well as information on required and optional parameters, HTTP status and error codes, and code samples you may use in your workflows.
+
+### New Endpoints
+
+The table below describes the new endpoints that have been added to the CircleCI API for this updated v2 version.
+
+Endpoint       | Description                       
+-----------|-------------------------------------------------------
+`GET /workflow/:id ` | This endpoint enables users to return an individual Workflow based on the `id` parameter being passed in the request
+`GET /workflow/:id/jobs` | This endoint enables users to retrieve all Jobs associated with a specific workflow, based on its unique `id`.
+`GET /project/:project_slug`  | This endpoint enables users to retrieve a specific Project by its unique slug.
+`POST /project/:project_slug/pipeline` | This endpoint enables users to retrieve an individual project by its unique slug.
+`GET /pipeline/:id` | This endpoint enables users to retrieve an individual pipeline, based on the `id` passed in the request.
+`GET /pipeline/:id/config`  | This endpoint enables users to retrieve the configuration of a specific pipeline.
+`GET /project/:project_slug/pipelines/[:filter]`  | This endpoint enables users to retrieve the most recent set of pipelines for a Project.
+
+### Deprecated Endpoints
+{:.no_toc}
+
+For this updated API v2 release, several endpoints have been deprecated, which are listed in the table below.
+
+Endpoint       | Description
+-----------|-----------------------------------------------------
+`POST /project/:vcs-type/:username/:project`  | This endpoint allowed users to trigger a new build.
+`POST /project/:vcs-type/:username/:project/build` | This endpoint enables users to trigger a new build by project.
+`DELETE /project/:vcs-type/:username/:project/build-cache` | This endpoint enabled users to clear the project cache for a specific project.
+`GET /recent-builds` | This endpoint enabled users to retrieve an array of recent builds.
+
+## API v2 and Server Customers
+
+Existing CircleCI Server customers will be unable to use API v2. Sometime in the future, support for Server customers may be added.
```

Comment:
I wonder if it might sound less bad to say: "API V2 is not currently supported for self-hosted installations of CircleCI Server."